fares-data-build-tool/create-data
# feat: reflect carnet tickets in netex

# Description

-  Allows for production of Carnet netex (single, return, flat fare, period geo zone, period multiple services)

# Testing instructions

-  Pull down this branch and rebuild from scrathc (this will update the matching json stored in local S3)#
-  From the dev repo, run 'make generate-validate-all-parallel' and ensure everythin validates.

# Type of change


# Checklist:
